src/components/ha-analytics.ts

```diff
@@ -0,0 +1,128 @@
+import { HomeAssistant } from "../types";
+import "./ha-icon-button";
+import { haStyle } from "../resources/styles";
+import type { HaCheckbox } from "./ha-checkbox";
+import { isComponentLoaded } from "../common/config/is_component_loaded";
+import { Analytics } from "../data/analytics";
+
+const preferences = ["base", "diagnostics", "usage", "statistics"];
+
+@customElement("ha-analytics")
+export class HaAnalytics extends LitElement {
+  @property({ attribute: false }) public hass!: HomeAssistant;
+
+  @property({ attribute: false }) public analytics!: Analytics;
+
+  protected render(): TemplateResult {
+    if (!this.analytics.huuid) {
+      return html``;
+    }
+    return html`
+      <p>
+        ${this.hass.localize(
+          "ui.panel.config.core.section.core.analytics.instance_id",
+          "huuid",
+          this.analytics.huuid
+        )}
+      </p>
+      ${preferences.map((preference) =>
+        preference === "base" || this.analytics.preferences.includes("base")
+          ? html`<ha-settings-row>
+              <span slot="prefix">
+                <ha-checkbox
+                  @change=${this._handleRowCheckboxClick}
+                  .checked=${this.analytics.preferences.includes(preference)}
+                  .preference=${preference}
+                >
+                </ha-checkbox>
+              </span>
+              <span slot="heading">
+                ${preference === "usage"
+                  ? isComponentLoaded(this.hass, "hassio")
+                    ? this.hass.localize(
+                        `ui.panel.config.core.section.core.analytics.preference.usage_supervisor.title`
+                      )
+                    : this.hass.localize(
+                        `ui.panel.config.core.section.core.analytics.preference.usage.title`
+                      )
+                  : this.hass.localize(
+                      `ui.panel.config.core.section.core.analytics.preference.${preference}.title`
+                    )}
+              </span>
+              <span slot="description">
+                ${preference === "usage"
+                  ? isComponentLoaded(this.hass, "hassio")
+                    ? this.hass.localize(
+                        `ui.panel.config.core.section.core.analytics.preference.usage_supervisor.description`
+                      )
+                    : this.hass.localize(
+                        `ui.panel.config.core.section.core.analytics.preference.usage.description`
+                      )
+                  : this.hass.localize(
+                      `ui.panel.config.core.section.core.analytics.preference.${preference}.description`
+                    )}
+              </span>
+            </ha-settings-row>`
+          : ""
+      )}
+      <p>
+        ${this.hass.localize(
+          "ui.panel.config.core.section.core.analytics.documentation",
+          "link",
+          html`<a href="https://www.home-assistant.io/integrations/analytics">
```

Comment:
Use docklink URL helper

Reply:
It is nice to see you reviewing your own PR's. I'll just leave you to it 👋